### DISCUSSION TITLE
How to download a non-event clip?

### DISCUSSION THREAD

**Discussion by ronisaacson** [blakeblackshear/frigate#6801] [Ask A Question] [answered]
When reviewing events, there's a download button for each clip in the event list. But I also have motion recording enabled and I'm watching a clip in the timeline view that I want to download. The clip contains motion but no detected objects so it doesn't show in the event list. I don't see a download button anywhere, so how do I get this clip?

**Comment by NickM-27** [comment, marked as answer]
For now you have to [use the api](https://docs.frigate.video/integrations/api#get-vodcamerastartstart-timestampendend-timestampindexm3u8). In the next version of frigate (0.13) there will be an export page to export whatever time rang you want as a standard recording or as a timelapse

**Reply by ronisaacson** [reply]
Great, thanks!

**Reply by AussiSG** [reply]
@NickM-27 Currently running 0.14.1 but when I'm trying to export a time line which has not been trigger by an event.
It gives the error there is no event to export
Is there another way to get that recording via export?

**Reply by NickM-27** [reply]
the export does not work via events, it will work as long as there are recordings for the timeframe requested

to help more we will need to see your exact api call and response

**Reply by AussiSG** [reply]
Oeh i'm not using the api call. I'm using the export function in frigate.
I thought you mentioned that it would be possible to export in version >= 0.13 even with out an event being triggered on that moment.
I've now made a screen recording haha.

What I noticed is that that recording frame has far less resolution.
Is it possible to get it with an higer resolution through the api call?

**Reply by NickM-27** [reply]
> What I noticed is that that recording frame has far less resolution.

exports always use the full resolution recordings when they are saved

**Reply by AussiSG** [reply]
Hmm when trying to export it through an API call it states that there is not recording at that timestamp.
Tho if I check frigate there is some footage present , not an event that was trigger but I do see footage of that moment.

Trying the following in my windows cmdline:
It is supposed to fetch some footage of the following specific time range.

from
Date and time (GMT): Wednesday, November 6, 2024 16:33:00 
to
Date and time (GMT): Wednesday, November 6, 2024 16:35:00 
```
curl -X POST http://192.168.0.12:5000/api/export/front/start/1730910780/end/1730910900
{"message":"No recordings found for time range","success":false}
```

**Reply by NickM-27** [reply]
> Tho if I check frigate there is some footage present , not an event that was trigger but I do see footage of that moment.

are you seeing actual recording footage or just the low res preview?

**Reply by AussiSG** [reply]
Oei I think only low-res preview.
I can scroll through the time line to see it.

But it is not high res like an triggers Event.

So I guess I won't be able to get an high-res normal video of that period

**Reply by AussiSG** [reply]
Would it be possible to download the low res video in some sort of way ?

**Reply by NickM-27** [reply]
it is in 0.15
